### PR TITLE
AnneJu step 9-1: Set up i18n for Japanese

### PR DESCRIPTION
**Pull request by anneju**
1. install gem i18n-rails and set config
2. create yml files for Japanese & English wordings
3. replace hard-coded wordings with i18n

Below is Japanese version. Sorry for my poor Japanese in advance~~ 🤖

**Index**
Screen Shot 2022-12-15 at 11 06 29

**New**
Screen Shot 2022-12-15 at 11 06 37

**Show**
Screen Shot 2022-12-15 at 11 06 50

**Edit**
Screen Shot 2022-12-15 at 11 06 57

### CODE REVIEW

File: myapp/config/application.rb [hunk 1]
```diff
@@ -22,5 +22,9 @@ class Application < Rails::Application
     config.generators do |generator|
       generator.fixture_replacement :factory_bot, suffix_factory: 'factory'
     end
+
+    config.i18n.default_locale = :en # change to :en back later so no need to rewrite test
```
Comment (anneju) [inline myapp/config/application.rb:26]: preferr to test in English 🙈 So I just check the display of Japanese version works fine~

File: myapp/config/locales/general/ja.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+ja:
+  helpers:
+    list: 一覧
+    show: 見る
```
Comment (yo-it-engineer) [inline myapp/config/locales/general/ja.yml]: [Optional]I think the more suitable word for japanese is「詳細」.

File: myapp/spec/system/tasks_spec.rb [hunk 3]
```diff
@@ -3,11 +3,15 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:submit_botton_text) { I18n.t('helpers.submit.submit', model: I18n.t('activerecord.models.task')).capitalize }
```
Comment (yo-it-engineer) [inline myapp/spec/system/tasks_spec.rb]: [spell check] button? bottom? 😄

**Review by yo-it-engineer — APPROVED** [review]
LGTM!!

IMG_1575

File: myapp/config/locales/general/ja.yml (hunk 2 above)
Comment (anneju) [reply myapp/config/locales/general/ja.yml]: yeap, that's better 🙈

File: myapp/spec/system/tasks_spec.rb (hunk 3 above)
Comment (anneju) [reply myapp/spec/system/tasks_spec.rb:6]: oops, typo~~

Comment (anneju) [reply myapp/spec/system/tasks_spec.rb:6]: button = ボタン ? No? the submit button~

Comment (anneju) [reply myapp/spec/system/tasks_spec.rb]: oops, typo~~ 😅

**Pull request merged**